.github/workflows/preview_on_pr.yml

```diff
@@ -59,7 +59,8 @@ jobs:
       with:
         schema: ${{ matrix.values.schema }}
         target: ${{ matrix.values.target }}
-        strict: '1'
+          #Uncomment to disable strict checks 
+          #no-strict: true 
```

Comment:
😂- I'm don't think we should disable the strict checking. People won't know when the yamls are not up to the spec

Reply:
The API of the workflow changed in v4. The _default_ is now `strict`. If we want non-strict, we need to uncomment the section that I commented out.

Reply:
ah okay! Just remove the commented out lines then!

Reply:
Done!